src/clusterfuzz/_internal/protos/generate.sh

```diff
@@ -36,7 +37,7 @@ read -r -d '' COPYRIGHT_HEADER <<EOF
 # limitations under the License.
 EOF
 
-for generated in $SCRIPT_DIR/*pb2*.py; do
+for generated in $SCRIPT_DIR/*pb2*.py{,i}; do
```

Comment:
What does this addittion do?

Reply:
mypy-protobuf generates type stubs as `.pyi` files, which need to be copied to the right place as well as the `_pb2.py` files.

Reply:
ACK